src/port1.0/portbump.tcl

```diff
@@ -0,0 +1,316 @@
+# et:ts=4
+# portbump.tcl
+#
+# Copyright (c) 2002 - 2004 Apple Inc.
+# Copyright (c) 2004 - 2005 Paul Guyot <pguyot@kallisys.net>
+# Copyright (c) 2006 - 2012, 2014 - 2016, 2018 The MacPorts Project
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+# 1. Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+# 2. Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+# 3. Neither the name of Apple Inc. nor the names of its contributors
+#    may be used to endorse or promote products derived from this software
+#    without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+
+package provide portbump 1.0
+package require portutil 1.0
+package require portchecksum 1.0
+
+set org.macports.bump [target_new org.macports.bump portbump::bump_main]
+target_provides ${org.macports.bump} bump
+target_runtype ${org.macports.bump} always
+target_requires ${org.macports.bump} main fetch
+target_prerun ${org.macports.bump} portbump::bump_start
+
+namespace eval portbump {
+}
+
+# bump_start
+#
+# Target prerun procedure; simply prints a message about what we're doing.
+#
+proc portbump::bump_start {args} {
+    global UI_PREFIX portpath
+
+    ui_notice "$UI_PREFIX [format [msgcat::mc "Bumping checksums for %s (Portfile --> Distfile)"] [option subport]]"
```

Comment:
What is the `(Portfile --> Distfile)` output supposed to mean?

Reply:
I just realised it's confusing, I was trying to explain that we're updating the portfile based on the new distfile. I'll remove it